### ISSUE TITLE
0.4.3.86 - perf counter - no status information

### ISSUE THREAD

**Issue by mathieuchateau** [mickem/nscp#76]
just to keep track of this issue :)

**Comment by mickem** [comment]
Great, I'll close the other one then...

**Comment by agherardi** [comment]
Not sure if this is the same problem. With 0.4.3.77, I'm seeing the following:

# /usr/lib/nagios/plugins/check_nrpe -H 192.168.240.24 -c CheckCounter -a "Counter:proc=\\Processor(_total)\\% Processor Time" ShowAll MaxWarn=10 MaxCrit=10

OK: |'proc_value'=22;0;0

**Comment by mickem** [comment]
First off "this issue" which is that the message only containes "ok" have yet to be fixed (will be fixed in the next release).
The other issue which is that you get _value on the perf data was fixed "after" build 77 so you need build 88.

**Comment by agherardi** [comment]
>>>> will be fixed in the next release
Thanks. By "next release" do you mean 0.4.4 or a newer 0.4.3 build?

**Comment by mickem** [comment]
Next build of 0.4.4 (wanted to the urgent ones for this weeks build, so expect nightly sometime this week, and an official release early next week)

**Comment by agherardi** [comment]
Excellent, thank you.

**Comment by mickem** [comment]
Fixed in next build, sorry for the delay but ended up busy last week.

```
D  w32system Created command: warn=value>=10 crit=value>=10 show-all "counter:proc=\Processor(_total)\% Processor Time" perf-config=*(suffix:none)
L        cli OK: OK: proc = 5
L        cli  Performance data: 'proc'=5;0;0
```

**Comment by mickem** [comment]
This is fixed in the nightly build 0.4.3.90
